packages/fxa-settings/src/components/BentoMenu/index.test.tsx

```diff
@@ -23,6 +25,7 @@ describe('BentoMenu', () => {
     fireEvent.click(toggleButton);
     expect(toggleButton).toHaveAttribute('aria-expanded', 'true');
     expect(screen.queryByTestId(dropDownId)).toBeInTheDocument();
+    testAllL10n(screen, bundle);
```

Comment:
I love that we can run this check now. I wonder how best to include it. I think what you have here is good, and we could just add it as needed. I do wonder, however, if this will get neglected over time.
 
Maybe putting this in the `afterEach` function would be worth it? This would encourage its usage. It seems like different tests will leave the UI in different states with different sets of l10n-ids on the page. Therefore running it after each test might give better coverage? I don’t have a good feel for how much overhead this would add though.

Reply:
Agreed, and placing in an `afterEach` sounds good to me. I was torn on if we should use `screen` here, I'd originally tried making it a little more agnostic so we could pass in a function pulled off the render function (e.g. `const { queryByTestId } = render` and figured if we ran per test we could refactor it to allow for that , but if it's in an `afterEach`, we'd need to use `screen` for sure, which in the end might make us more consistent anyway, so long as folks are cool with using a global. I'll change this in that follow up issue.